src/sentry/static/sentry/app/views/projectsDashboard/chart.tsx

```diff
@@ -2,27 +2,32 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
 
+import {Project} from 'app/types';
 import BarChart from 'app/components/barChart';
 
-export default class Chart extends React.Component {
-  static propTypes = {
-    stats: PropTypes.array,
-  };
+type Props = {
+  stats: Required<Project>['stats'];
+};
 
-  static defaultProps = {
-    stats: [],
-  };
+const Chart = ({stats}: Props) => {
+  const data = stats.map(d => ({x: d[0], y: d[1]}));
 
-  render() {
-    const data = this.props.stats.map(d => ({x: d[0], y: d[1]}));
+  return (
+    <div>
+      <StyledBarChart points={data} label="events" height={60} gap={1.5} />
+    </div>
+  );
+};
 
-    return (
-      <div>
-        <StyledBarChart points={data} label="events" height={60} gap={1.5} />
-      </div>
-    );
-  }
-}
+Chart.propTypes = {
+  stats: PropTypes.array,
+};
+
+Chart.defaultProps = {
+  stats: [],
```

Comment:
Should we move it to the stats parameter?

Reply:
whoops thought i got both of them